python/README.md

```diff
@@ -12,71 +10,23 @@ $ pip install foxglove-websocket
 
 ## Example server
 
-The following script provides a simple example server which publishes messages on a single topic called `example_msg`, using JSON to encode message data and [JSON Schema](https://json-schema.org) to describe the message layout.
-
-To get started with the example server:
+Run a simple example server that publishes messages on a single `example_msg` topic, using JSON to encode message data and [JSON Schema](https://json-schema.org/) to describe the message layout:
 
-1. Copy the script below into a file named `example.py` and run `python3 example.py`.  
-   **or:** Install the foxglove-websocket package and run `python -m foxglove_websocket.examples.simple_server`.
+```
+python -m foxglove_websocket.examples.simple_server
+```
 
-2. In a browser, open up https://studio.foxglove.dev and initiate a Foxglove WebSocket connection to `ws://localhost:8765/`.
+To see data from this server, open [Foxglove Studio](https://studio.foxglove.dev?ds=foxglove-websocket&ds.url=ws://localhost:8765/) with a Foxglove WebSocket connection to `ws://localhost:8765/`:
 
 <img width="676" alt="Foxglove Studio displaying data from the example server" src="https://user-images.githubusercontent.com/14237/145260376-ddda98c5-7ed0-4239-9ce4-10778ee8240b.png">
 
-```py
-import asyncio
-import json
-import time
-from foxglove_websocket import run_cancellable
-from foxglove_websocket.server import FoxgloveServer, FoxgloveServerListener
-from foxglove_websocket.types import ChannelId
-
-
-async def main():
-    class Listener(FoxgloveServerListener):
-        def on_subscribe(self, server: FoxgloveServer, channel_id: ChannelId):
-            print("First client subscribed to", channel_id)
+## Server template
```

Comment:
Not sure this section is worth keeping anymore. It just points at a script that we already explained in the `Example server` section.

Reply:
Combining with previous section (in-line links to files).